public/scripts/group-chats.js

```diff
@@ -1309,11 +1330,13 @@ function getGroupCharacterBlock(character) {
     const isFav = character.fav || character.fav == 'true';
     template.data('id', character.avatar);
     template.find('.avatar img').attr({ 'src': avatar, 'title': character.avatar });
-    template.find('.ch_name').text(character.name);
-    template.attr('chid', characters.indexOf(character));
+    template.find('.ch_name').text(character.name + (character.queueOrder > 0?'   (#' + character.queueOrder + ')':''));    template.attr('chid', characters.indexOf(character));
     template.find('.ch_fav').val(isFav);
     template.toggleClass('is_fav', isFav);
+    template.toggleClass('is_active', character.queueOrder === 1);
+    template.toggleClass('is_queued', character.queueOrder > 1);
     template.toggleClass('disabled', isGroupMemberDisabled(character.avatar));
+    template.toggleClass('is_active', character.avatar === (this_chid && characters[this_chid] && characters[this_chid].avatar));
```

Comment:
Why are you toggling `is_active` class twice here? Which one is the correct? Should likely be merged.

Reply:
Because I'm not very smart :)  Fixed in latest commit